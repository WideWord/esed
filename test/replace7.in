aad
ewf 
ewf
fre
ferfgrtgrt g
rtg
rt
g rtgrtgertg rt
gtr
grteher;hklme
rhrtphmrthrh'rthkekrh
rteh'rmthreh;rtkmer
ert;hrnherth;jklehnlernth
erwglhjkberg
memerwg ew
w erge