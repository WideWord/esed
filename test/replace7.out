aad
ewf 
ewf
fre
ferfgrtgrt g
rtg
rt
g rtgrtgertg rt
gtr
grteher;hklme
rhrtphmrthrh'rthmemrh
rteh'rmthreh;rtkmer
ert;hrnherth;jklehnlernth
erwglhjkberg
memerwg ew
w erge


awdqwdd e
wf
ewfewfewew fwefewf ewf weq qwefewqf ewfwfweqfewfeqw eqwl fqwelf ewqfqwefpqwelf 
qwe fpqweflewpf ,ewqfpqe,fepwqfqw,e fpweq ,feqwp qewf,epwfeqw,pef,qw p
eqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofeweqw feqwfpoeqwfpweqfqwpofew